components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -6,7 +6,7 @@
 
 import { FunctionComponent, useCallback } from "react";
 import ContextMenu, { ContextMenuEntry } from "../components/ContextMenu";
-import { OrgIcon, OrgIconProps } from "../components/org-icon/OrgIcon";
```

Comment:
**thought**: Hope changes here don't affect dedicated onboarding. How could I test this? 🤷